docs/en/stack/ml/df-analytics/ml-supervised-workflow.asciidoc

```diff
@@ -0,0 +1,142 @@
+include, but is not limited to, mitigating redundancy, reducing biases, applying 
+standards and/or conventions, data normalization, and so on.
+
+{regression-cap} and {classification} requires specifically structured source 
+data: a two dimensional tabular data structure. For this reason, you might need 
+to {ref}/transforms.html[{transform}] your data to create a {dataframe} which 
+can be used as the source for {dfanalytics} like {regression} and 
+{classification}.
+
+[[train-test-iterate]]
+== Train, test, iterate
+
+After your data is prepared and transformed into the right format, it is time to 
+train the model. Training is an iterative process — every round of iteration is 
+followed by an evaluation to see how the model performs. When you are satisfied 
+with the results, you are ready to deploy the model otherwise you may want to 
+adjust the training configuration or consider alternative ways to preprocess and 
+represent your data.
+
+The first step is defining the features – the relevant fields in the data set – 
+that will be used for training the model. By default, all the fields with 
+supported types are included in classification and regression automatically. 
+However, you can exclude irrelevant fields optionally from the process. Doing so 
+makes a large dataset more manageable and by reducing the computing resources 
+and time required for training.
+
+Now define how to split your training data, using what's called the train-test 
+split. There is no optimal percentage that fits for all use cases, it depends on 
+the amount of data and the time you have to train. The test set won’t be used to 
+train the model, but only for testing during the learning process.
+
+During the training process, the training data that contains the ground truth 
+you want to learn from is fed through the learning algorithm. The model predicts 
+the value and compares it to the ground truth then the model is fine-tuned to 
+make the predictions more accurate.
+
+Once the model has been trained, it will be tested on  how well it predicts for 
+previously unseen data. This test provides an estimate of a quantity known as 
+the model generalization error. There are further evaluation types for both 
+{classification} and {regression} analysis which provide metrics about training 
+performance. 
+
+
+[[deploy-model]]
+== Deploy model
+
+You have trained the model and are satisfied with the performance. The last step 
+is to deploy your trained model and start using it on new data.
+
+Trained models are stored as {es} documents in an internal index. The Elastic 
+{ml} feature called {infer} enables you to use the model in a continuous fashion 
+either by using it as a processor in an ingest pipeline, in a continuous 
+{transform} or as an aggregation at search time. When new data comes into your 
+ingest pipeline or you run a search on your data with an {infer} aggregation, 
+the model is used to infer against the data and make predictions on it.
```

Comment:
It seems out of place to mention an implementation detail such as `Trained models are stored as {es} documents in an internal index.`

Phase `enables you to use the model in a continuous fashion` doesn't feel quite right. Inference enables you to make predictions for new data. This could be done thru ingest, continuous transforms or an agg.